### PR TITLE
Get rid of infinity sign in U/D

### PR DESCRIPTION
**Pull request by Dukobpa3**
I've just made this PR but then figured out that we don't need the infinity sign at all.
We can see not loaded torrents by many others marks like empty bar or status icon

https://github.com/openscopeproject/TrguiNG/pull/135

Maybe not merge it for now because I trying to understand how to provide custom sorting for columns with `accessorFn`.
Most of them sorting in some unpredictable way

Could you help with some manuals?
I'm not coding ts at all, just raw js a little. My main langs is java/c#.

### CODE REVIEW

**Comment by qu1ck** [comment]
What is displayed in the field has no effect on how it is sorted if it has accessorFn. Only the value that the accessor returns effect sorting order. For infinity case the accessor likely returns NaN which javascript sorts however it sorts, if you want to change sorting order you need to take care of 0 download case separately.
That said, you can't just remove infinity without replacement and I doubt there is a better one.

**Comment by Dukobpa3** [comment]
So with sorting I think the issue is that All accessorFn's returns '<div></divv>'
For U/D I've just made custom one sorting.

**Comment by Dukobpa3** [comment]
I've removed inifinity just because of cleaner look and code)) Not because of sorting

**Comment by Dukobpa3** [comment]
@qu1ck I've fixed sorting.
So if U ok with it – ready to merge

**Comment by qu1ck** [comment]
Sorting fix looks good, but keep the infinity.

**Comment by Dukobpa3** [comment]
Done

**Comment by Dukobpa3** [comment]
Wait then

I'l play with it a little

Wanna get proper sorting between inf and zero

File: src/components/tables/torrenttable.tsx [hunk 1]
```diff
@@ -93,7 +93,12 @@ const AllFields: readonly TableField[] = [
         name: "uploadedEver",
         label: "U/D",
         component: UploadRatioField,
-        accessorFn: (t) => t.uploadedEver / t.downloadedEver,
+        accessorFn: (t) =>
+            t.downloadedEver === 0
+                ? -1
+                : t.uploadedEver === 0
+                    ? 0
+                    : t.uploadedEver / t.downloadedEver,
```
Comment (Dukobpa3) [inline src/components/tables/torrenttable.tsx:101]: @qu1ck 
Move it to function or leave as is?

**Comment by qu1ck** [comment]
I fixed it so infinity is not below zero in sorting and 0 upload always shows "-".
Thanks for contributing!

**Comment by Dukobpa3** [comment]
And as result you've made only dashes))
We will never see infinity now. I like it (:

**Comment by qu1ck** [comment]
Infinity is still shown when upload is > 0 and download is 0, as was intended. That is the case when you are seeding data that you already had, for example uploading your own torrent.

**Comment by Dukobpa3** [comment]
Sorting still not perfect but I think there is enough time wasted already))

**Pull request merged**